packages/opc-feedback/README.md

```diff
@@ -71,50 +71,56 @@ import '@one-platform/opc-feedback/dist/opc-feedback';
 ## Properties
 | Name   | value |
 |---------|------------------------------------------------------------------|
-| ```url``` | ```String``` value is expected ```/feedback``` is the default fallback url. |
+| ```spaUrl``` | ```String``` value is expected ```/feedback``` is the default fallback url. |
+| ```docUrl``` | ```String``` value is expected ```/get-started``` is the default fallback url. |
```

Comment:
@riginoommen Can you please create a `validation function` for these checks in the code?
It should do a console.warn when the user provides incorrect values.

Reply:
Included as suggested